### ISSUE TITLE
getxtrade.com

### ISSUE THREAD

**Issue by llUnknownl** [Discord-AntiScam/scam-links#99]
**URL**
getxtrade.com

**Additional context**
A user hacked was spamming a YouTube short (and steamcommuxity links, but you already have this in the filter) on my server, I entered to investigate and it was a user impersonating Elon Musk advertising this website.
It was just created 10 days ago and it's design is really similar to previous crypto scam websites, I'm sure you will notice it.

I already reported this to the registrar abuse contact but I submit thos anyways.

Thanks for the good work!

**Comment by SoFnSkittles** [comment]
Yeah I got that same video and I went to the website and did the little sign up put in the code and I got my 5.25 ethereum and right now that should have sitting at like $15,000 but here is the caveat to it you can't withdraw anything from this fucking website under any circumstances whatsoever anytime you try to validate yourself through their methodology you get to level two and then you get stuck they never validate you on level three and then when you click withdrawal the only way to withdraw is to validate yourself by making a purchase of Bitcoin through there services and when you do that you send the Bitcoin to an address that you have absolutely no idea of where it's going to or who it's going to but they say that they have cold wallets and hot wallets that are set up for this exact thing but if you go and check the aetherium or bit address that they provide to you none of them have ever had any trades sales buys anything done on them whatsoever literally stay away from this shit unless you want to lose money because they're going to ask you for between $200 and $250,300 of Bitcoin ethereum Litecoin whatever and they want you to purchase it through their site and send it to an address that they provide to you that has absolutely no history to it whatsoever here I even have the screenshots to prove all of it you people need to watch out for shit like this because they're trying their ass off to get everybody nowadays
![Screenshot_20240219-105426](https://github.com/Discord-AntiScam/scam-links/assets/97749214/ec9cf834-d75d-45a8-8b52-ee2f9ce4cf45)
![Screenshot_20240219-105439](https://github.com/Discord-AntiScam/scam-links/assets/97749214/55a90e2c-8cf5-4d75-9bc7-32a47c2805ec)
![Screenshot_20240219-105749](https://github.com/Discord-AntiScam/scam-links/assets/97749214/79c6ae48-f7e2-4d69-865c-b1ace4d37caf)
![Screenshot_20240219-105447](https://github.com/Discord-AntiScam/scam-links/assets/97749214/0d86e2e4-c1d0-4718-8ddc-f77bf8e8845d)
![Screenshot_20240219-105506](https://github.com/Discord-AntiScam/scam-links/assets/97749214/346559f9-1393-44ce-8587-bd223eae46e3)
![Screenshot_20240219-105809](https://github.com/Discord-AntiScam/scam-links/assets/97749214/1cbeee6d-166c-4abb-9040-3faf12c9cf47)

There's all the proof that you should ever need this is bullshit and they are trying to do nothing but take your initial validation payment and you'll never see it again so make sure you protect yourself and you are on top of all of these scams because these assholes are doing everything they can to take everything you have.

**Comment by SoFnSkittles** [comment]
And I got a message this morning when I logged into it that everybody who has money in their account and does not make their validation payment will be reset back to zero in 48 hours if they have not made their validation payment anybody want to tell me that's not a Ftc  violation

**Comment by actually-akac** [comment]
Hello, thank you for the detailed report about this domain. We've added the domain into our database in commit https://github.com/Discord-AntiScam/scam-links/commit/0a2c1d2437d51aa81efe9c4d278da7df9d49c342.
